pkg/util/cgroup/cgroup.go

```diff
@@ -233,10 +233,6 @@ func getCgroupDetails(mountInfoPath string, cRoot string, controller string) (mo
 	scanner := bufio.NewScanner(info)
 	for scanner.Scan() {
 		fields := bytes.Fields(scanner.Bytes())
-		if len(fields) < 10 {
```

Comment:
also line 303

Reply:
No, this is not part of the fix, just a duplication.

We can remove it if the later we will filter it again.